pySDC/tests/test_projects/test_AC/test_simple_forcing.py

```diff
@@ -15,15 +15,15 @@ def test_main_parallel():
     import mpi4py
 
     nprocs = 2
-    cmd = f"export PYTHONPATH=$PYTHONPATH:$(pwd); mpirun -np {nprocs} python pySDC/projects/AllenCahn_Bayreuth/run_simple_forcing_benchmark.py -n {nprocs}"
+    cmd = f"export PYTHONPATH=$PYTHONPATH:$(pwd); export HWLOC_HIDE_ERRORS=2; mpirun -np {nprocs} python pySDC/projects/AllenCahn_Bayreuth/run_simple_forcing_benchmark.py -n {nprocs}"
```

Comment:
What's the matter here?

Reply:
So the tests won't fail on OpenSuse systems (the ones we have to deal with at TUHH ...)

Reply:
Ah, ok, thanks. Didn't know that!